site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -141,7 +141,7 @@ See the link:/docs/v1/tech/oauth/endpoints#logout[Logout endpoint] for more info
 [field]#Logout behavior# [optional]#Optional#::
 This selector allows you to modify the behavior when using the link:/docs/v1/tech/oauth/endpoints#logout[Logout endpoint] with this Client Id.
 +
-    * All applications - This is the default behavior. Logout out of the FusionAuth SSO, call each registered Logout URLs for the entire tenant and then redirect to the Logout URL registered for this application.
+    * All applications - This is the default behavior. Upon Logout of the FusionAuth SSO, call each registered Logout URLs for the entire tenant and then redirect to the Logout URL registered for this application.
```

Comment:
I noticed the redundant "Logout out" when I was looking at logout behavior stuff.